### PR TITLE
feat(rc): Add Remote Config Parameter Value Type Support

### PR DESCRIPTION
**Pull request by lahirumaramba**
go/admin-sdk-rc-parameter-value-types

- Add RC Parameter Value Type.
- Update unit tests.
~- Integration tests will be added following the REST API launch.~
- Added `release:stage` to trigger existing integration tests (to test backward compatibility).
~- `Do not merge` until the BE is updated.~

Update:
- Integration tests are updated.

RELEASE NOTE: Added Remote Config Parameter Value Type Support.

### CODE REVIEW

File: test/unit/remote-config/remote-config.spec.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ describe('RemoteConfig', () => {
             'android_en': { value: 'A Droid must love a pumpkin spice latte.' },
           },
           description: 'Description of the parameter.',
+          valueType: 'STRING' as remoteConfig.ParameterValueType,
```
Comment (hiranya911) [inline test/unit/remote-config/remote-config.spec.ts:54]: Where is this used?

**Review by hiranya911 — APPROVED** [review]
LGTM with a question.

**Review by egilmorez — APPROVED** [review]
The two doc strings look good to gilmorez, thanks!

File: test/unit/remote-config/remote-config.spec.ts (hunk 1 above)
Comment (lahirumaramba) [reply test/unit/remote-config/remote-config.spec.ts:54]: This is used in `REMOTE_CONFIG_RESPONSE` in line 98 and also be used in the parameter groups tests (ex: line 506)

**Pull request merged**